src/test/java/uk/gov/digital/ho/pttg/application/namematching/candidates/MultipleLastNamesFunctionsTest.java

```diff
@@ -20,11 +19,10 @@
 
     @Test
     public void addAllLastNameCombinationsMultipleNames() {
-        List<CandidateName> candidateNames = new ArrayList<>();
         List<String> firstNames = Arrays.asList("Aaa", "Bbb", "Ccc");
         List<String> lastNameCombinations = Arrays.asList("Ddd", "Eee", "Ddd Eee");
 
-        List<CandidateName> newCandidateNames = addAllLastNameCombinations(candidateNames, firstNames, lastNameCombinations);
+        List<CandidateName> newCandidateNames = addAllLastNameCombinations(firstNames, lastNameCombinations);
```

Comment:
As above - now that this method doesn't alter a list passed in as a parameter its name should change.

Reply:
Actually now I've read the test, it wasn't altering the list but the name is still poor.